docs/cpp/faq-cpp.md

```diff
@@ -9,11 +9,11 @@ MetaDescription: Frequently asked questions about the C/C++ extension in Visual
 ---
```

Comment:
Could you update the Date so we know we already have done the freshness pass?

Reply:
Okay, I wasn't sure who is supposed to update that date.

Reply:
No problem, as far as I am aware it can be anyone on the team, does not have to be Greg or Tyler, so you are good to go.